docs/components/camera/_index.md

```diff
@@ -173,6 +173,55 @@ For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/c
 {{% /tab %}}
 {{< /tabs >}}
 
+### GetImages
+
+Get simultaneous images from different sensors, along with associated metadata.
+This should not be used for getting a time series of images from the same sensor.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- None.
+
+**Returns:**
+
+- [(List[NamedImage])](https://python.viam.dev/autoapi/viam/media/video/index.html#viam.media.video.NamedImage): The list of images returned from the camera system.
+- [(ResponseMetadata)](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.ResponseMetadata): The metadata timestamp with this response.
+
+```python {class="line-numbers linkable-line-numbers"}
+my_camera = Camera.from_robot(robot=robot, name="my_camera")
+
+frame = await my_cam.get_images()
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/camera/index.html#viam.components.camera.Camera.get_images).
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+
+**Returns:**
+
+- [([]image.Image)](https://pkg.go.dev/image#Image): The list of images returned from the camera system.
+- [(time.Time)](https://pkg.go.dev/time#Time): The metadata timestamp with this response.
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
```

Comment:
These have recently been updated in Go

```
- [([]camera.NamedImage)](https://github.com/viamrobotics/rdk/blob/5486114bf2bb29975c328242a86239216b3f73b2/components/camera/camera.go#L66): The list of images returned from the camera system, with the name of the sensor associated with the image.
- [(resource.ResponseMetadata)](https://github.com/viamrobotics/rdk/blob/5486114bf2bb29975c328242a86239216b3f73b2/resource/response_metadata.go#L10): The metadata, which holds timestamp of when the data was captured.
- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
```

Reply:
Ok - let's wait until those updates are in the go docs then